packages/horizon-add-document/handler.js

```diff
@@ -9,15 +9,10 @@ const config = {
   },
 };
 
-async function parseFile(log, documentId, applicationId, documentType) {
-  log.info({ applicationId, documentId, documentType }, 'Downloading document');
-
-  const { data } = await axios.get(`/api/v1/${applicationId}/${documentId}/file`, {
-    baseURL: config.documents.url,
-    params: {
-      base64: true,
-    },
-  });
+function createDataObject(data, body) {
+  const { documentType } = body;
+  const documentInvolvement = body.documentInvolvement || 'Document:Involvement';
+  const documentGroupType = body.documentGroupType || 'Document:Document Group Type';
```

Comment:
What about if they're in the body but have "" - does this also set the default value if the key exists but is null?